package.json

```diff
@@ -48,8 +48,14 @@
 		"typedoc-plugin-coverage": "3.4.0",
 		"typedoc-plugin-mdn-links": "4.0.4",
 		"typescript": "catalog:",
-		"typescript-eslint": "^8.18.0",
-		"vite": "catalog:",
+		"typescript-eslint": "^8.18.1",
+		"vite": "^6.0.3",
```

Comment:
Is moving from `catalog:` related to the necessity to include vite `v6` in overrides?

Reply:
Yes, it seems that pnpm update automatically rewrites it.